Clients/GUI/Pages/SignUpWizzard.razor

```diff
@@ -0,0 +1,126 @@
+﻿@page "/signup"
+@using System.IO
+@using GUI.Components
+@using GUI.Scripts
```

Comment:
I think we already have this in imports

Reply:
delete some using, but I can't delete GUI.Components